lib/subscriber_list_mover.rb

```diff
@@ -69,12 +69,12 @@ def email_change_to_subscribers(source_subscriber_list)
 
     bulk_move_template = <<~BODY.freeze
       Hello,
-
-      You've subscribed to get email alerts about #{list_title} information.
-
-      GOV.UK is changing the type of email alerts you get to make sure you find out when any #{list_title} guidance is added or updated.
-      You can [change how often you get updates or unsubscribe](#{email_redirect}) from GOV.UK email alerts.
-
+      You've subscribed to get emails about #{list_title}.
+      
+      GOV.UK is changing the way we send emails, so you may notice a difference in the number and type of updates you get.
+      
+      If you're getting too many emails
+      You can [manage your subscription](#{email_redirect}) to choose how often you want to receive emails.
```

Comment:
Looks like we've got an Grammar issue here. I'm not sure if `If you're getting too many emails` is missing the end of a sentence or if the "You" is meant to be lowercase?

Reply:
Ah it was supposed to be a heading!